lib/vagrant/action/general/package.rb

```diff
@@ -83,6 +84,20 @@ def call(env)
           compress
         end
 
+        def output_inside_folder?(env)
+          env["package.output"].include?('/')
+        end
+
+        def package_with_folder_path(fullpath)
+          folder_path = File.expand_path('../', fullpath)
```

Comment:
I think this can just be `File.expand_path('..', fullpath)` so that it works on windows.